app/assets/stylesheets/header.css

```diff
@@ -0,0 +1,28 @@
+.account-actions {
```

Comment:
Good work on css!
Let's also study the concept of [BEM](https://getbem.com/) and use it accordingly.